src/Symfony/Component/HttpKernel/Controller/ControllerResolver.php

```diff
@@ -167,4 +169,15 @@ protected function instantiateController($class)
     {
         return new $class();
     }
+
+    private function getArgumentResolverManager()
+    {
+        if (null === $this->argumentResolverManager) {
+            $this->argumentResolverManager = new ArgumentResolverManager();
```

Comment:
What about adding constructor with optional argument `$resolvers` and pass instances of resolvers to them instead of calling `add` by hand twice here?